### DISCUSSION TITLE
how can I create those options with radio buttons?

### DISCUSSION THREAD

**Discussion by DevAdrian02** [serenity-is/Serenity#6421] [Help]
I want to create a survey in the system, I would like to add radio buttons, but I can't do it, I would like to know how to get this data from the DB.

PST: how can I create those options with radio buttons?

**Comment by VictorTomaili** [comment]
Hello @DevAdrian02,
Please don't spam on other issues for your question. If anyone knows your answer, they will answer here. This is community support discussion area for questions and knowledge sharing. Issues are just for bugs since discussions are enabled.
